src/Tooltip/Tooltip.component.tsx

```diff
@@ -4,20 +4,9 @@ import * as React from 'react';
 import classNames from 'classnames';
 import styled, { css } from '@xstyled/styled-components';
 import { th } from '@xstyled/system';
-
-export type Position =
-    | 'topStart'
-    | 'top'
-    | 'topEnd'
-    | 'rightStart'
-    | 'right'
-    | 'rightEnd'
-    | 'bottomEnd'
-    | 'bottom'
-    | 'bottomStart'
-    | 'leftEnd'
-    | 'left'
-    | 'leftStart';
+// ANCHOR
+import { positionVariants, Position } from '../utils/position/position';
+export { Position } from '../utils/position/position';
```

Comment:
It was the fastest way to extract the position utility without having to redo imports from this file.

But ultimately I figured it was alright to leave it here and be able to import from this file to see what positions exist. I don't feel strongly about it however, just seems more convenient.

Reply:
Since you're doing it more for convenience in the developing of the stories I'm not going to be picky about it. My concern was more about the ultimate export of two components, if `Position` was being exported twice that could be a problem. As that's not what's going on, I'm not worried.